docs/modules/ROOT/pages/documentation/extensions/notifiers/README.adoc

```diff
@@ -0,0 +1,29 @@
+= Notifier Extensions
+
+The notifier extensions allow notifications from the Notification Service to propagate to various channels on top of the notifiers provided by default.
+
+You can find the code for each notifier here: https://github.com/payara/Notifiers. That codebase includes an example notifier that can be used as a base for any additional notifiers you might want.
+
+== Notifier Summary
+
+Each notifier extension is composed of 2 parts:
+
+### The Core Module
+
+The core module contains all the code necessary to receive notifications from the notification service and propagate them to their proposed channel. When installing this module, you'll get the admin commands to configure the notifier, as well as the ability to select that notifier in the notification sources such as Request Tracing commands and admin console pages.
+
+### The Admin Console Plugin
+
+The admin console plugin contains all the code necessary to configure the notifier in the admin console. This is exclusively the tab in the notification service page. This module is not required for the notifier to function, but makes administrating the notifier easier.
+
+== Installing a Notifier
+
+To install a notifier extension: download the artifacts listed on the notifier documentation page and drop them into the `${GLASSFISH_HOME}/modules` directory of your Payara Server installation. Alternatively, you can build the module straight from the Github repository and drop all JARs from the `target/` directory of the extension into the same installation directory.
+
+You can then restart the server to see your extension activated.
+
+TIP: Make sure that if the notifier has dependency JARs in order to run, that those dependencies are also in the installation directory.
```

Comment:
I think this tip is a bit too vague, especially with users who are not acquainted on how OSGI/HK2 work in Payara Server, so it can be confusing for users to know how to manage dependencies for extensions